sdk/resourcemanager/Azure.ResourceManager/src/autorest.md

```diff
@@ -269,6 +269,14 @@ directive:
   - rename-model:
       from: ProviderRegistrationRequest
       to: ProviderRegistrationOptions
+  - from: resources.json
+    where: $.definitions.Alias
+    transform:
+      $["x-ms-client-name"] = "ResourceTypeAlias";
```

Comment:
In the comment all `alias*` need to be prepend model name, so here it changed that only fix the `alias` ?